pkg/services/searchV2/types.go

```diff
@@ -14,21 +14,22 @@ type FacetField struct {
 }
 
 type DashboardQuery struct {
-	Query        string       `json:"query"`
-	Location     string       `json:"location,omitempty"` // parent folder ID
-	Sort         string       `json:"sort,omitempty"`     // field ASC/DESC
-	Datasource   string       `json:"ds_uid,omitempty"`   // "datasource" collides with the JSON value at the same leel :()
-	Tags         []string     `json:"tags,omitempty"`
-	Kind         []string     `json:"kind,omitempty"`
-	PanelType    string       `json:"panel_type,omitempty"`
-	UIDs         []string     `json:"uid,omitempty"`
-	Explain      bool         `json:"explain,omitempty"` // adds details on why document matched
-	Facet        []FacetField `json:"facet,omitempty"`
-	SkipLocation bool         `json:"skipLocation,omitempty"`
-	AccessInfo   bool         `json:"accessInfo,omitempty"` // adds field for access control
-	HasPreview   string       `json:"hasPreview,omitempty"` // the light|dark theme
-	Limit        int          `json:"limit,omitempty"`      // explicit page size
-	From         int          `json:"from,omitempty"`       // for paging
+	Query              string       `json:"query"`
+	Location           string       `json:"location,omitempty"` // parent folder ID
+	Sort               string       `json:"sort,omitempty"`     // field ASC/DESC
+	Datasource         string       `json:"ds_uid,omitempty"`   // "datasource" collides with the JSON value at the same leel :()
+	Tags               []string     `json:"tags,omitempty"`
+	Kind               []string     `json:"kind,omitempty"`
+	PanelType          string       `json:"panel_type,omitempty"`
+	UIDs               []string     `json:"uid,omitempty"`
+	Explain            bool         `json:"explain,omitempty"`            // adds details on why document matched
+	WithAllowedActions bool         `json:"withAllowedActions,omitempty"` // adds allowed actions per entity
+	Facet              []FacetField `json:"facet,omitempty"`
+	SkipLocation       bool         `json:"skipLocation,omitempty"`
+	AccessInfo         bool         `json:"accessInfo,omitempty"` // adds field for access control
```

Comment:
I added this flag "accessInfo" perhaps prematurely...  thinking it would eventually do exactly what withAllowedActions is doing here :)